cf_xarray/tests/test_accessor.py

```diff
@@ -1233,3 +1233,53 @@ def test_cmip6_attrs():
     )
     assert da.cf.axes["X"] == ["nlon"]
     assert da.cf.axes["Y"] == ["nlat"]
+
+
+def test_custom_criteria():
+    my_custom_criteria = {
+        "ssh": {
+            "standard_name": (
+                "sea_surface_height_amplitude_due_to_geocentric_ocean_tide_geoid_mllw",
+                "sea_surface_height_above_sea_level_geoid_mllw",
+            ),
+            "name": ("sea_surface_elevation",),  # variable name
+            "units": ("m",),
```

Comment:
I don't see how `units` could ever work here. With the current logic, everything with units of "m" will match `ds.cf["ssh"]`. That's not right.

Reply:
True, removed units.